metabase/metabase
# Remove outdated ExpressionPopover

Relates to https://github.com/metabase/metabase/issues/29029

### Description

Remove outdated ExpressionPopover as it was listed in TODO.
Reroute usages to ExpressionWidget.

### How to verify

1. Start creating a new Question: click "New" button in the top bar -> Question.
2. Click "Add filters to narrow your answer" in the Notebook -> Select "Custom Expression" -> Appeared expression editor should have new design and experience.
3. Close Filters popover
4. Click "Pick the metric you want to see" in Summarize section -> Select "Custom Expression" -> Appeared expression editor should have new design and experience.

### Demo


https://user-images.githubusercontent.com/7983744/225456766-ce212a85-ec36-4eba-a1d5-e26473135ce8.mov



### Checklist

- [x] Refine visuals (width for expression editor)


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29050)
